chronicle/release/format/markdown/presenter.go

```diff
@@ -71,7 +82,7 @@ func (m Presenter) formatChangeSections(changes change.Changes) string {
 			result += formatChangeSection(section.Title, summaries) + "\n"
 		}
 	}
-	return result
+	return strings.TrimRight(result, "\n")
```

Comment:
Why do we now need to remove trailing newlines?

Reply:
Now there is a footer and it became noticeable that we have a large amount of whitespace before the footer -- this makes the whitespace between the content and footer smaller